content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -193,9 +193,11 @@ spec:
           - registry_service_name: istio-ingressgateway.istio-system.svc.cluster.local
             port: 443
 
-  # Change the Istio service `type=LoadBalancer` and add the cloud provider specific annotations. See
+  # Change the Istio service `type=LoadBalancer` add the cloud provider specific annotations. See
   # https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer for more
   # information. The example below shows the configuration for GCP/GKE.
+  # If the GCP/GKE version is less than 1.16, service_annotations needs to add network.gke.io/internal-load-balancer-allow-global-access: "true" parameter.
+  # https://stackoverflow.com/questions/59680679/gcp-internal-load-balancer-global-access-beta-annotation-does-not-work?answertab=active#tab-top
```

Comment:
```suggestion
  # If the GCP/GKE version is less than 1.16, add `network.gke.io/internal-load-balancer-allow-global-access: "true"`to the `service_annotations`.
  # See https://stackoverflow.com/questions/59680679/gcp-internal-load-balancer-global-access-beta-annotation-does-not-work?answertab=active#tab-top.
```

Reply:
I have made changes to the PR, please review again. tks!